src/app/modules/recipe-form/components/recipe-view/recipe-view.component.ts

```diff
@@ -0,0 +1,74 @@
+import {Component, Input, OnInit, Output, EventEmitter} from '@angular/core';
+import {Emitted, Recipe} from "../../models/recipe-create.model";
+import {RecipeFormService} from "../../services/recipe-form.service";
+import {ActivatedRoute, Router} from "@angular/router";
+
+@Component({
+  selector: 'recipe-view',
+  templateUrl: './recipe-view.component.html',
+  styleUrls: ['./recipe-view.component.scss']
+})
+export class RecipeViewComponent implements OnInit {
+
+  @Input() form_name: string[] = ['',''];
+  @Input() currentRecipe?: Recipe;
+  @Output() newRecipeEvent = new EventEmitter<Emitted>();
+
+  newRecipe: Recipe =
+    {id: 0, name: '', content: '', video_link: '', price: '', cooking_time: 1, level: ''};
+  splitContent: string[] = [];
+  currentId?: number;
+
+  constructor(private route: ActivatedRoute,
+              private router: Router,
+              private recipeFormService: RecipeFormService) {
+  }
+
+  private contentPrep(): string {
```

Comment:
always try to use the full name instead of shortcuts.
```suggestion
  private prepareContent(): string {
```